src/euroeval/dataset_configs/estonian.py

```diff
@@ -13,3 +13,20 @@
     task=SENT,
     languages=[ET],
 )
+
+WINOGRANDE_ET_CONFIG = DatasetConfig(
+    name="winogrande_et",
+    pretty_name="the Estonian Winogrande dataset",
```

Comment:
To maintain consistency with other datasets, we include the task name as well here:

```suggestion
    pretty_name="the Estonian common-sense reasoning dataset Winogrande-et",
```